PPAK_CVS/Sources/Storage/Storage.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Storage.swift
+//  PPAK_CVS
+//
+//  Created by 김응철 on 2022/12/25.
+//
+
+import Foundation
+
+final class Storage {
+  
+  static let shared = Storage()
+  
+  private let key = "Storage"
+  private let userDefaults = UserDefaults.standard
+  private init() {}
+  
+  private(set) var products: [ProductModel] = shared.load() {
+    didSet {
+      let encoder = JSONEncoder()
+      if let encoded = try? encoder.encode(oldValue) {
+        userDefaults.set(encoded, forKey: key)
+      }
+    }
+  }
+  
+  func load() -> [ProductModel] {
```

Comment:
위에서 `load()`메서드를 내부 메서드로 처리했으므로, 이 녀석도 private로 선언하는 것이 좋을 것 같아요.
```suggestion
  private func load() -> [ProductModel] {
```